pkg/skaffold/schema/latest/config.go

```diff
@@ -23,19 +23,28 @@ import (
 
 const Version string = "skaffold/v1beta7"
 
-// NewSkaffoldPipeline creates a SkaffoldPipeline
+// NewSkaffoldPipeline creates a SkaffoldConfig
 func NewSkaffoldPipeline() util.VersionedConfig {
-	return new(SkaffoldPipeline)
+	return new(SkaffoldConfig)
 }
 
-// SkaffoldPipeline describes a Skaffold pipeline.
-type SkaffoldPipeline struct {
+// SkaffoldConfig describes a Skaffold pipeline configuration file.
```

Comment:
wording is a little weird here, maybe something like 

`SkaffoldConfig holds the fields parsed from the Skaffold configuration file`?